app/globals.css

```diff
@@ -2,6 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
+body {
+  background-color: #555555;
```

Comment:
Could you apply this within the className? You want to be as consistent as possible throughout your entire application. Also, please define any hardcoded values in your tail.config.